apps/next-evans/pages/audiobooks.tsx

```diff
@@ -0,0 +1,124 @@
+import { getDocumentUrl, getFriendUrl } from '@/lib/friend';
+import { getAllDocuments } from '@/lib/db/documents';
+import { isNotNullish } from '@/lib/utils';
+import { formatDuration, formatMonthDay, newestFirst } from '@/lib/dates';
+import { LANG } from '@/lib/env';
+import Audiobook from '@/components/pages/audiobooks/Audiobook';
+
+export const getStaticProps: GetStaticProps<Props> = async () => {
+  const sortedBooks = Object.values(await getAllDocuments())
+    .filter((book) => book.hasAudio)
+    .filter(isNotNullish)
+    .sort(newestFirst);
+  return {
+    props: {
+      numAudioBooks: 7,
+      sortedBooks,
+    },
+  };
+};
+
+interface Props {
+  numAudioBooks: number;
+  sortedBooks: Array<
+    Doc<'editions' | 'authorGender' | 'shortDescription' | 'mostModernEdition'>
+  >;
+}
+
+const AudioBooks: React.FC<Props> = ({ numAudioBooks, sortedBooks }) => (
+  <div className="">
+    <AudiobooksHero numBooks={numAudioBooks} className="pb-52" />
+    <div className="bg-flgray-200 py-12 xl:pb-6">
+      <Dual.H2 className="sans-wider text-center text-2xl md:text-3xl mb-12 px-10">
+        <>Recently Added Audio Books</>
+        <>Audiolibros añadidos recientemente</>
+      </Dual.H2>
+      <div className="flex flex-col xl:flex-row items-center justify-center gap-12 xl:gap-28 px-12 xl:px-28 xl:pl-36">
+        {sortedBooks.slice(0, 2).map((book) => {
+          const audiobook = book.mostModernEdition.audiobook;
+          invariant(audiobook);
+          return (
+            <BookTeaserCard
+              {...book}
+              editions={book.editions}
+              htmlShortTitle={htmlShortTitle(book.title)}
+              documentUrl={getDocumentUrl(book)}
+              authorUrl={getFriendUrl(book.authorName, book.authorGender)}
+              description={book.shortDescription}
+              key={getDocumentUrl(book)}
+              className="xl:-ml-8"
+              badgeText={formatMonthDay(audiobook.createdAt, LANG)}
+              audioDuration={formatDuration(audiobook.duration)}
+            />
+          );
+        })}
+      </div>
+    </div>
+    <div>
+      <div className="pt-24">
+        <h2 className="sans-wider text-center text-3xl mb-8 px-8">{t`All Audio Books`}</h2>
+        <Dual.P className="body-text text-center text-lg mb-12 px-8 sm:px-10">
+          <>
+            Browse all {sortedBooks.length} audiobooks below, or{` `}
+            <Link href={t`/audio-help`} className="subtle-link">
+              get help with listening here
+            </Link>
+            .
+          </>
+          <>
+            Explora los {sortedBooks.length} audiolibros a continuación u{` `}
+            <Link href={t`/audio-help`} className="subtle-link">
+              obtén ayuda aquí para escuchar
+            </Link>
+            .
+          </>
+        </Dual.P>
+      </div>
+      <div className="flex gap-x-8 gap-y-36 flex-wrap justify-center py-24 px-4 sm:px-20">
```

Comment:
two things here:

1. at `xl` the current production site shows **3** columns of audio books, but you're only showing 2. was this intentional?
2. check the deploy previews, the text of the buttons on this component is not correct.